pilot/pkg/server/server.go

```diff
@@ -56,6 +56,7 @@ import (
 	"istio.io/istio/pilot/pkg/serviceregistry"
 	"istio.io/istio/pilot/pkg/serviceregistry/aggregate"
 	"istio.io/istio/pilot/pkg/serviceregistry/cloudfoundry"
+	"istio.io/istio/pilot/pkg/serviceregistry/cluster"
```

Comment:
Please don't just call it `cluster`. Envoy clusters and k8s clusters are already confusing to use in the same place.

Reply:
There really isn't a better name than `cluster`. This file doesn't make any reference to Envoy clusters, so there is no ambiguity.